enterprise/utils.py

```diff
@@ -177,7 +177,8 @@ def get_oauth2authentication_class():
     try:
         from openedx.core.lib.api.authentication import OAuth2AuthenticationAllowInactiveUser as OAuth2Authentication
     except ImportError:
-        from edx_rest_framework_extensions.auth.bearer.authentication import BearerAuthentication \
+        # Temporary until OAuth2Authentication has been moved out of edx-platform
+        from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication \
```

Comment:
For reviewer, this seemed like the easiest short term solution to removing BearerAuthentication in this case.

Reply:
Within edx-enterprise this is only called once for cornerstone it looks like.  Are there other libraries that are also calling this function?